### ISSUE TITLE
Possible conflict of settings?

### ISSUE THREAD

**Issue by domitreous** [Macjutsu/super#105]
First, thanks for this wonderful piece of work! 
Quick question, do these two settings create a conflict? They both appear on paper to deal specifically in security patching and I have noticed some failures from super to reboot the Mac when having both enabled. 

Configuration profile setting:
<key>AllowRSRUpdates</key>
<true/>

Apple auto updates config profile setting
"automatically install system data files and security updates"

**Comment by Macjutsu** [comment]
These two settings do not conflict with each other...

AllowRSRUpdates = `super` will not ignore RSR updates (the default behavior is to ignore them), but instead ask `softwareupdate` to install them. This function literally only controls `super`'s workflow.

Apple "Automatically install..." = tell `softwareupdated` to automatically install any security updates if it finds them during the occasional automatic check. This function literally only controls `softwareupdated`'s automatically scheduled workflow.

In other words each only controls TIMING of when an RSR might install automatically... neither setting can "prevent" an RSR from being installed.

Also if your referencing this week's RSR... Apple horked it up badly on their own... https://www.macrumors.com/2023/07/10/apple-pulls-ios-16-5-1-macos-13-4-1-rsrs/
